acquia_lift.module

```diff
@@ -1497,6 +1511,33 @@ function acquia_lift_library() {
     ),
   );
 
+  // The Acquia Lift debugger.
+  $libraries['debugger'] = array(
+    'title' => 'Acquia Lift debugger',
+    'website' => '',
+    'version' => VERSION,
+    'js' => array(
+      $path . '/js/acquia_lift.debug.js' => $options,
+      'https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.2/angular.min.js' => $options,
```

Comment:
This should be done as a library dependency (similar to how we do backbone and underscore).  We probably need to add this to the drush make file as well as to our requirements checks.

Pretty crazy that we'll now have a requirement for angular, backbone, underscore, and other smaller libraries.  How about we leave this as an external include for now but then add a @todo to migrate to a library dependency.